growth/too/models.py

```diff
@@ -301,35 +301,44 @@ def gracedb(self):
     def ned_gwf(self):
         return "https://ned.ipac.caltech.edu/gwf/events"
 
+    def get_property(self, property_name, value=None):
```

Comment:
Prefix the name of this function with an underscore to indicate that it is not a part of the public interface of this class.